### ISSUE TITLE
ADD: Crousel to Homage which will increase user interactivity 

### ISSUE THREAD

**Issue by shubhagarwal1** [Bitbox-Connect/Bitbox#36]

**Comment by Anuj3553** [comment]
Hyy @shubhagarwal1, Give me some sample then i will assign you the issue.

**Comment by shubhagarwal1** [comment]
> Hyy @shubhagarwal1, Give me some sample then i will assign you the issue.

okay so on the website on the landing page for now here in the below image these divs look very much not appeallign to the user also acts like too much at once 
![Screenshot 2024-10-09 at 9 50 22 PM](https://github.com/user-attachments/assets/446bd8aa-68c9-402c-b637-c26926cde90b)
will enhae this part and add crousel to it

**Comment by Anuj3553** [comment]
Okk @shubhagarwal1, I'm assigning you this issue. All the best.

**Comment by shubhagarwal1** [comment]
Hey @Anuj3553 can you please help me with project setup

**Comment by shubhagarwal1** [comment]
hey @Anuj3553 can you, are you active, can you help me with the project setup, i am struglling with the project set up issue

**Comment by Anuj3553** [comment]
> hey @Anuj3553 can you, are you active, can you help me with the project setup, i am struglling with the project set up issue

Hello @shubhagarwal1, What do you need help?

**Comment by shubhagarwal1** [comment]
> > hey @Anuj3553 can you, are you active, can you help me with the project setup, i am struglling with the project set up issue
> 
> 
> 
> Hello @shubhagarwal1, What do you need help?

Hey @Anuj3553 can we connect over discord, i am troubling with project set up in prev. msgs i have described the prblm i am facing

**Comment by Anuj3553** [comment]
> > > hey @Anuj3553 can you, are you active, can you help me with the project setup, i am struglling with the project set up issue
> > 
> > 
> > Hello @shubhagarwal1, What do you need help?
> 
> Hey @Anuj3553 can we connect over discord, i am troubling with project set up in prev. msgs i have described the prblm i am facing

Yes you can connect with me any social platforms. Go to my github profile.
